internal/cmd/kafka/confluent_kafka.go

```diff
@@ -274,8 +274,11 @@ func RunConsumer(consumer *ckafka.Consumer, groupHandler *GroupHandler) error {
 }
 
 func (h *GroupHandler) RequestSchema(value []byte) (string, map[string]string, error) {
+	if len(value) == 0 || value[0] != 0x0 {
+		return "", nil, errors.New("unknown magic byte")
```

Comment:
Should I add a suggestion here? Maybe something about incorrect message formatting?

Reply:
Yes! I'm not sure what "unknown magic byte" means to be honest

Reply:
It's a value used to validate the file format.